traffic_portal/app/src/common/modules/table/serverServerCapabilities/table.serverServerCapabilities.tpl.html

```diff
@@ -25,7 +25,7 @@
             <li class="active">Capabilities</li>
         </ol>
         <div class="pull-right">
-            <button type="button" name="addCapabilityBtn" class="btn btn-primary" title="Add Server Capability" ng-click="addServerCapability()"><i class="fa fa-plus"></i></button>
+            <button ng-show="isCache(server)" type="button" name="addCapabilityBtn" class="btn btn-primary" title="Add Server Capability" ng-click="addServerCapability()"><i class="fa fa-plus"></i></button>
```

Comment:
i think ng-if would be fine as the server type will not be changing in this view